.github/pull_request_template.md

```diff
@@ -0,0 +1,50 @@
+<!---
+  This template provides some ideas of things to include in your PR description.
+  To start, try providing a short summary of your changes in the Title above.
+  If a section of the PR template does not apply to this PR, then delete that section.
+ -->
+
+## Motivation
+
+_(fill-in or delete this section)_
+
+<!--- 
+  What goal is this change working towards?
+  If this PR fixes one of more issues, list them here.
+  One line each, like so:
+    Fixes #123
+    Fixes #39
+-->
+
+## Changes
```

Comment:
Do we want to add another section for the release notes, like:
`## Does this PR introduce a user-facing change?`

Reply:
Great idea! I'll add that

Reply:
✅  https://github.com/urfave/cli/pull/963/commits/dad06805995ca192ca92eb4a3b657257a4f16a29